snapcraft/plugins/copy.py

```diff
@@ -57,6 +64,6 @@ def build(self):
             os.makedirs(os.path.dirname(dst), exist_ok=True)
 
             if os.path.isdir(src):
-                shutil.copytree(src, dst)
+                shutil.copytree(src, dst, symlinks=self.options.follow_symlinks)
```

Comment:
@femdom good catch, I missed that switch.

@sergiusens, @elopio we could use your input on the symlinks potentially reaching out of the .snap. Should we make sure that doesn't happen with a smarter copy function, or allow them to shoot themselves in the foot if they don't want to follow symlinks?